com.ca.lsp.cobol/lsp-core-domain/src/test/java/com/broadcom/lsp/domain/cobol/databus/impl/DataBusGetFromCacheBadTest.java

```diff
@@ -78,7 +76,6 @@ public void setUp() {
   @After
   public void tearDown() {
```

Comment:
I think we don't really need this method due to we do not startup a DI context. If you delete it, please, do it also for the other tests in this suite.